github.com.hashicorp.go.kms.wrapping.v2.types.proto

```diff
@@ -143,6 +143,9 @@ message Options {
   KeyEncoding with_wrapped_key_encoding = 80;
 
   bool with_disallow_env_vars = 90;
+
+  // WithNoHmac specifies that an HMAC is not necessary for the mechanism, even if marked as "required"
+  bool with_no_hmac = 100;
```

Comment:
s/with_no/without?  I don't feel strongly either way.

Reply:
sgtm. that's a more natural phrasing.